esp/src/src/Utility.ts

```diff
@@ -473,6 +473,9 @@ export function resolve(hpccWidget, callback) {
         case "ECLPlaygroundWidget":
             require(["hpcc/ECLPlaygroundWidget"], doLoad);
             break;
+        case "ECLArchiveWidget":
+            require(["src/ECLArchiveWidget"], doLoad);
+            break;
         case "ECLSourceWidget":
             require(["hpcc/ECLSourceWidget"], doLoad);
             break;
```

Comment:
Shouldn't this case have been removed?

Reply:
No - I ported the code into TypeScript (but left logic alone).